pkg/cmd/server/apis/config/types.go

```diff
@@ -924,6 +924,10 @@ type IdentityProvider struct {
 	MappingMethod string
 	// Provider contains the information about how to set up a specific identity provider
 	Provider runtime.Object
+	// LocalGroups determines if groups asserted by this provider should be prepended.
+	// If false, all groups asserted by this provider are prepended with the provider's name + "/"
+	// If true, all valid groups are used as-is without any modification.
+	LocalGroups bool
```

Comment:
this field needs a better name

Reply:
UnqualifiedGroups

Reply:
Sounds reasonable, I will see if anyone else has suggestions before updating.